specification/applicationinsights/ApplicationInsights.LiveMetrics/main.tsp

```diff
@@ -0,0 +1,42 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-autorest";
+import "@azure-tools/typespec-azure-core";
+import "./routes.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+@doc("QuickPulse REST APIs.")
+@versioned(Versions)
+@useAuth(
+  ApiKeyAuth<ApiKeyLocation.header, "x-ms-qps-auth-api-key"> | OAuth2Auth<[
+    {
+      type: OAuth2FlowType.implicit,
+      authorizationUrl: "https://login.microsoftonline.com/common/oauth2/authorize",
+      scopes: ["user_impersonation"],
+    }
+  ]>
+)
+@service({
+  title: "QuickPulse REST APIs for client SDKs.",
+})
+@server(
+  "{endpoint}",
+  "Existing QuickPulse global endpoint.",
+  {
+    @doc("The endpoint of the QuickPulse service.")
+    endpoint: string = "https://rt.services.visualstudio.com",
+  }
+)
+namespace LiveMetrics;
```

Comment:
I believe this should be `Azure.{Topic}.LiveMetrics` where the topic (or group) is one of those listed here: <https://azure.github.io/azure-sdk/general_design.html#general-namespaces-approved-list>